spec/requests/jobs_pages_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "JobsPages", type: :request do
+  describe "GET /jobs_pages" do
+    before do
+      @user = create(:user)
+      allow_any_instance_of(ApplicationController)
+        .to receive(:current_user).and_return(@user)
```

Comment:
I don't know how to fix this....it keeps giving me a different error for each fix I do